src/shork.gleam

```diff
@@ -146,13 +185,24 @@ pub fn query(sql: String) -> Query(Nil) {
   Query(sql:, parameters: [], row_decoder: decode.success(Nil))
 }
 
+pub type TransactionError {
+  TransactionQueryError(QueryError)
+  TransactionRolledBack(String)
+}
```

Comment:
There are a bunch of public functions and types with no documentation yet - is that on purpose?

Reply:
I am not done yet documenting them